client/src/app/tabs/cloud-bpmn/BpmnEditor.js

```diff
@@ -142,6 +146,13 @@ export class BpmnEditor extends CachedComponent {
 
     propertiesPanel.attachTo(this.propertiesPanelRef.current);
 
+
+    try {
+      await this.loadTemplates();
```

Comment:
What's the reason of blocking the `checkImport` until we load templates?

Reply:
I was wondering the same, and checked out where it was added: https://github.com/camunda/camunda-modeler/pull/1069/files#diff-8d663070a9293c2317ca1a7dff4fa8a7004cbd6855d29d128635aad4fa20d67cR107-R113

This was back when we moved to React, so probably this is even older. If I would guess, I'd say to avoid setting the templates from the config while the Modeler crashes on import? Maybe it's related to the old properties panel implementation, but I couldn't say that we can safely remove it.

Reply:
OK, let's leave it as is.